django/core/management/utils.py

```diff
@@ -173,3 +174,56 @@ def run_formatters(written_files, black_path=(sentinel := object())):
             [black_path, "--fast", "--", *written_files],
             capture_output=True,
         )
+
+
+SPACE_PREFIX = "    "
+BRANCH_PREFIX = "│   "
+TEE_PREFIX = "├── "
+ELBOW_PREFIX = "└── "
+
+
+def get_directory_tree(path, max_depth=10, prefix="", depth=0):
+    """
+    Get the folder structure of a directory and it's content,
+    akin to the GNU `tree` command.
+    Adapted from: https://stackoverflow.com/a/59109706
+
+    Example:
+    >>> for line in get_directory_tree(Path.home()):
+    ...     print(line)
+    """
+
+    path = Path(path)
```

Comment:
Is this needed? I’m surprised by this as it seems we’d be calling `get_directory_tree` with `Path` instances anyway, and in your example above we’re also calling it with a `Path`.